src/Tribe/Views/V2/Rest_Endpoint.php

```diff
@@ -224,11 +246,12 @@ public function get_mocked_rest_request( array $params ) {
 	 * and use the same method behind the scenes to make sure we have consistency.
 	 *
 	 * @since  4.9.7
-	 *
-	 * @return void
+	 * @since TBD Look up the POST data before the GET one to process the request.
 	 */
 	public function handle_ajax_request() {
-		$request = $this->get_mocked_rest_request( $_GET );
+		// Use the POST method data, if set; else fallback on the GET data.
+		$source = isset( $_POST ) ? $_POST : $_GET;
```

Comment:
* :warning: Equals sign not aligned with surrounding assignments; expected 2 spaces but found 1 space
* :no_entry_sign: Processing form data without nonce verification.
* :no_entry_sign: Processing form data without nonce verification.
* :warning: Processing form data without nonce verification.

<sub>hash: de57cc50f8ca156487c6240e30c93ec6</sub>